### PR TITLE
chore: Update example app

### PR DESCRIPTION
**Pull request by HW13**
### Summary 

 - Update example app to `await fastify.register(...)`
 - Update README and example app's OpenTelemtry SDKs to use latest packages
   - `@opentelemetry/tracing` -> `@opentelemetry/sdk-trace-base`
   - `@opentelemetry/node` -> `@opentelemetry/sdk-trace-node`
   - `TraceIdRatioBasedSampler` now exported from `@opentelemetry/sdk-trace-base`
 
### Test plan

 - Run the example app locally
 - Navigate to `http://localhost:3000` in the browser
 - Confirm spans are logged as expected

### CODE REVIEW

**Comment by HW13** [comment]
#50 will resolve the failing Validate workflow

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ npm i @autotelic/fastify-opentelemetry
 // to fastify-opentelemetry. (See the example configuration below.)
 require('./openTelemetryConfig')
 const openTelemetryPlugin = require('@autotelic/fastify-opentelemetry')
-const fastify = require('fastify')()
+const fastify = require('fastify')();
```
Comment (CLSnazel) [inline README.md:18]: Is the `;` meant to be here in this code example?

**Review by CLSnazel — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HW13) [reply README.md:18]: Because the next line of code is an IIFE, we need the semi-colon to prevent it from being treated as a function argument.

**Review by matthew-charles-chan — APPROVED** [review]
LGTM 💪

**Pull request merged**